### PR TITLE
Replaces glass in atmos tank windows with reinforced plasma glass

### PR DESCRIPTION
**Pull request by ACCount12**
It's way too easy for some random shit to vent them, and dealing with complete loss of all gas tanks isn't fun. 

This doesn't affect possibility of atmos sabotage, unless the saboteur was too dumb to bring some tools.

### CODE REVIEW

**Comment by CPTANT** [comment]
Boooo, smashing tanks is a legit response to plasma flooding when the situation is not under control.

**Comment by ACCount12** [comment]
Better bring some tools then. This PR if more of a fauna/random degenerates protection. If you are venting tanks with a purpose, it wouldn't stop you.

**Comment by ShizCalev** [comment]
You can still smash these windows, they just take more hits.

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -102671,6 +102671,34 @@
 	},
 /turf/open/floor/circuit/green,
 /area/engine/supermatter)
+"YGM" = (
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: Not required, but if you wouldn't mind could you condense these down to one key?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -81046,6 +81046,34 @@
 	},
 /turf/closed/wall/mineral/titanium,
 /area/shuttle/abandoned)
+"EDo" = (
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Ditto

**Review by ShizCalev — APPROVED** [review]

**Comment by ACCount12** [comment]
I'd better wait until the map merger itself is fixed. It's supposed to do that, not offload the work to humans.

**Pull request merged**